File: app/src/main/java/im/bernier/petfinder/model/Photo.java
package im.bernier.petfinder.model;

import org.simpleframework.xml.Attribute;
import org.simpleframework.xml.Element;
import org.simpleframework.xml.Root;
import org.simpleframework.xml.Text;

/**
 * Created by micha on 2016-07-09.
 */

@Root(name = "photo", strict = false)
public class Photo {

    @Attribute
    private int id;

    @Attribute
    private String size;

    @Text
    private String value;

    public int getId() {
        return id;
    }

    public void setId(int id) {
        this.id = id;
    }

    public String getSize() {
        return size;
    }

    public void setSize(String size) {
        this.size = size;
    }

    public String getValue() {
        return value;
    }

    public void setValue(String value) {
        this.value = value;
    }

    @Override
    public String toString() {
        return "Photo{" +
                "id=" + id +
                ", size='" + size + '\'' +
                ", value='" + value + '\'' +
                '}';
    }
}

File: app/src/main/java/im/bernier/petfinder/model/Photo.kt
/*
 *   This Source Code Form is subject to the terms of the Mozilla Public
 *   License, v. 2.0. If a copy of the MPL was not distributed with this
 *   file, You can obtain one at http://mozilla.org/MPL/2.0/.
 *
 * If it is not possible or desirable to put the notice in a particular
 * file, then You may include the notice in a location (such as a LICENSE
 * file in a relevant directory) where a recipient would be likely to look
 * for such a notice.
 *
 * You may add additional accurate notices of copyright ownership.
 */

package im.bernier.petfinder.model

import org.simpleframework.xml.Attribute
import org.simpleframework.xml.Root
import org.simpleframework.xml.Text

/**
 * Created by micha on 2016-07-09.
 */

@Root(name = "photo", strict = false)
data class Photo(@field:Attribute var id: Int = 0, @field:Attribute var size: String? = "", @field:Text var value: String? = "")